gpjax/kernels/nonstationary/linear.py

```diff
@@ -17,7 +17,9 @@
 
 import jax.numpy as jnp
 import tensorflow_probability.substrates.jax.bijectors as tfb
-from jaxtyping import Array, Float
+from gpjax.typing import Array
+from jaxtyping import Float
+from gpjax.typing import ScalarFloat
```

Comment:
Can we merge lines 20 and 22 please? I see this occurs for most of the kernel modules.

Reply:
Yeah, that's because I didn't want to touch two dozen files by hand and used `sed` for fixing the imports. I would prefer _not_ to merge those lines manually (a lot more effort because it's not always exactly the same so can't as easily write a `sed` line for it), instead, I would suggest tools like `isort` for consistent import order (similar to `black` for code formatting). But I would leave that for a separate PR that does nothing else other than "visual" changes (so e.g. one can ignore it in git blame).

Reply:
Yep, on `docs_update` that included in the pre-commit, so let's leave that job for then.